docker/app/Dockerfile

```diff
@@ -0,0 +1,15 @@
+from node:lts-alpine
+
+WORKDIR /home/www
+
+ADD package.json ./package.json
+RUN npm install
```

Comment:
I don't mind switching to `npm` over `yarn` (may not have been obvious). Not sure if this should be `npm ci` so it preserves a "working state" between deploys. I know that newer npm is fairly performant, so maybe just removing the `yarn` lock and switching to npm is going to be the smoothest.

Reply:
I don't have a strong preference here... For now updated to `npm`, however if you'd prefer to stick with yarn I can remove this change and make that one.